app/views/layouts/_navbar.html.haml

```diff
@@ -48,6 +48,3 @@
             =link_to events_path do
               %i.fa.fa-dashboard
               %span Dashboard
-
-- if params[:controller].include?("staff") && user_signed_in?
-  = render partial: "layouts/nav/staff_subnav"
```

Comment:
It doesn't work when I put the private methods in the Staff::ApplicationController - when I do that I get an undefined method error. However when I put it in the regular ApplicationController everything works and is still scoped properly.